src/com/cursor/Executor.java

```diff
@@ -0,0 +1,16 @@
+package com.cursor;
+import com.cursor.models.ConsoleObserver;
+import com.cursor.models.FileObserver;
+import com.cursor.models.WeatherStation;
+
+public class Executor {
+    public static void start() {
+        WeatherStation station = new WeatherStation();
+
+        station.addObserver(new ConsoleObserver());
+        station.addObserver(new FileObserver());
+
+        station.setMeasurements(25, 760);
+        station.setMeasurements(-5, 745);
+    }
+}
```

Comment:
do you see these red signs in the end? it's good practice to leave one empty line in the end of the file. that's just for further, not for now